src/services/azureFunctionsService.ts

```diff
@@ -68,16 +68,34 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 			return;
 		}
 		let projectFile = await azureFunctionUtils.getAzureFunctionProject();
+		let newHostProjectFile: azureFunctionUtils.IFileFunctionObject;
+		let hostFile: string;
+
 		if (!projectFile) {
 			let projectCreate = await vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened,
 				LocalizedConstants.createProject, LocalizedConstants.learnMore);
 			if (projectCreate === LocalizedConstants.learnMore) {
 				vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(constants.sqlBindingsDoc));
+				return;
 			} else if (projectCreate === LocalizedConstants.createProject) {
-				// start the create azure function project flow
-				await azureFunctionApi.createFunction({});
+				try {
+					// start the create azure function project flow
+					// create a watcher to see if the host file is created after the flow is complete
+					newHostProjectFile = await azureFunctionUtils.waitForNewHostFile();
+					const timeoutForHostFile = timeoutPromise(LocalizedConstants.azureFunctionsProjectMustBeOpened);
```

Comment:
You should create this after createFunction, otherwise you'll have the same bug we fixed before if the user takes a while to go through the createFunction flow